contrib/ansible/roles/kubernetes-addons/templates/skydns-rc.yaml.j2

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1beta3
```

Comment:
I wish we could re-use these, but i'm getting the sale v ansible/jinja variables make that impossible?